File: Conexion base de datos/C++/PostgreSQL.cpp
#include <iostream>
#include <libpq-fe.h>

int main() {
    // Cadena de conexión (similar a tus parámetros Python)
    const char* conninfo = "dbname=postgres user=postgres password=123456 host=localhost";
    
    // Establecer conexión
    PGconn* conn = PQconnectdb(conninfo);
    
    // Verificar conexión exitosa
    if (PQstatus(conn) != CONNECTION_OK) {
        std::cerr << "Error de conexión: " << PQerrorMessage(conn);
        PQfinish(conn);
        return 1;
    }
    
    std::cout << "✅ Conexión exitosa a PostgreSQL" << std::endl;
    
    // Cerrar conexión al finalizar
    PQfinish(conn);
    return 0;
}

File: Conexion base de datos/PHP/PostgreSQL.php
<?php
$host = "localhost";
$dbname = "postgres";
$user = "postgres";
$password = "123456";

try {
    // Cadena de conexión
    $conn = new PDO("pgsql:host=$host;dbname=$dbname", $user, $password);
    
    // Configurar el manejo de errores
    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
    
    echo "✅ Conexión exitosa a PostgreSQL";
    
} catch (PDOException $e) {
    echo "❌ Error de conexión: " . $e->getMessage();
}
?>